src/CMakeLists.txt

```diff
@@ -305,6 +305,13 @@ if(WITH_SEASTAR)
   endmacro ()
   set(Seastar_HWLOC OFF CACHE BOOL "" FORCE)
   set(Seastar_STD_OPTIONAL_VARIANT_STRINGVIEW ON CACHE BOOL "" FORCE)
+  if(Seastar_DPDK)
```

Comment:
would it make sense for ceph to set Seastar_DPDK=ON when WITH_DPDK=ON?

Reply:
i don't think so. currently, WITH_DPDK is solely for enabling the messenger's DPDK backend, which is not actively maintained anymore.

while crimson is still at its experiment stage and DPDK (native networking stack) is only interesting for those who want to evaluate the performance of crimson+dpdk.

they are quite different settings. i'd rather keep them this way. and in future, once crimson is GA, i think we will drop WITH_DPDK option.